src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -102,24 +85,22 @@ private RefAPISource() { /* To prevent instantiation */ }
      * Footnote: queries larger than pageSize will be truncated.
      *
      * @param pipelineOptions -- are used to get the credentials necessary to call the Genomics API
-     * @param apiData - contains the hashmap that maps from reference name to Id needed for the API call.
      * @param interval - the range of bases to retrieve.
      * @return the reference bases specified by interval and apiData (using the Google Genomics API).
      */
-    public ReferenceBases getReferenceBases(final PipelineOptions pipelineOptions, final RefAPIMetadata apiData, final SimpleInterval interval) {
+    public ReferenceBases getReferenceBases(final SimpleInterval interval, final PipelineOptions pipelineOptions) {
```

Comment:
Out of curiosity: was there a reason to swap the order of the arguments? This is going to be confusing because it doesn't match the order in e.g. buildReferenceNameToIdTable.